budcamp/Homework3/Homework3/task2/CaseTest.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework3
+{
+    internal class CaseTest: Test
+    {
+        public CaseTest Type { get; set; }
+        public string[] Variants { get; set; }
+
+        public CaseTest(int identifier, string topic, string question, string correctAnswer, 
+            CaseTest type, string[] variants): 
+            base(identifier, topic, question, correctAnswer)
+        {
+            this.Type = type;
+            this.Variants = variants;
+        }
+
+        public override void PrintTest()
+        {
+            Console.WriteLine(Topic);
+            Console.WriteLine();
+            Console.WriteLine(Question);
+            Console.WriteLine();
+            Console.WriteLine($"a) {Variants[0]}");
```

Comment:
Why have you assumed that your `Variants` always has exactly 4 elements in it???

I think what you wanted to do is:

```
for(int i = 0; i < Options.Length; i++)
{
    Console.WriteLine($"{i}) {Options[i]}");
}
```